packages/paste-style-props/src/border.ts

```diff
@@ -0,0 +1,43 @@
+export const BORDER_RADIUS_PROPS = {
```

Comment:
These are all based on the typings. So in the same way you can compose groups of these to nullify different design primtives.